specification/communication/data-plane/CallAutomation/preview/2024-09-01-preview/examples/CallRecording_GetRecordingResult.json

```diff
@@ -0,0 +1,39 @@
+{
+  "parameters": {
+    "endpoint": "https://contoso.communications.azure.com",
+    "api-version": "2024-09-01-preview",
+    "recordingId": "eyJQbGF0Zm9ybUVuZHBvaW50SWQiOiI4YjlkYmFkMS0xOGFiLTRmODYtYWQ0Yy1jNzVmODA5ODA3NmUiLCJSZXNvdXJjZVNwZWNpZmljSWQiOiI4YjlkYmFkMS0xOGFiLTRmODYtYWQ0Yy1jNzVmODA5ODA3NmUifQ=="
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "recordingStorageInfo": {
+          "recordingChunks": [
+            {
+              "documentId": "0-eus-d12-801b3f3fc462fe8a01e6810cbff729b8",
+              "index": 0,
+              "endReason": "SessionEnded",
```

Comment:
Is the service using PascalCase or camelCase? The swagger says camelCase but here it's PascalCase